src/libs/ReportActionsUtils.ts

```diff
@@ -372,7 +372,7 @@ function shouldReportActionBeVisible(reportAction: OnyxEntry<ReportAction>, key:
         return false;
     }
 
-    if (isPendingRemove(reportAction)) {
+    if (isPendingRemove(reportAction) && !isThreadParentMessage(reportAction, '')) {
```

Comment:
I don't really like this pattern; is there any other way we can figure out if it's a parent message without using this and passing an empty string? Alternately can we grab the reportID from somewhere?

Reply:
Here's [old discussion](https://github.com/Expensify/App/pull/28002#discussion_r1338165320)

Reply:
At the very least we should add a comment here to explain this, though I think I'd still prefer it if we could find a different way to check for this

Reply:
I think that works! I'd just be sure to add a comment explaining it above the function